rclpy/rclpy/node.py

```diff
@@ -1645,3 +1645,65 @@ def assert_liveliness(self) -> None:
         """
         with self.handle as capsule:
             _rclpy.rclpy_assert_liveliness(capsule)
+
+    def _get_info_by_topic(
+            self,
+            topic_name: str,
+            no_mangle: bool,
+            func: Callable[[object, str, bool], List[Dict]]) -> List[Dict]:
```

Comment:
@jaisontj @mm318 see [line 1545](https://github.com/ros2/rclpy/pull/454/files#diff-129fafcf34261ae2f52bd43718815308R1545) for formatting.

Reply:
Done.